deltachat-jsonrpc/src/api/mod.rs

```diff
@@ -39,6 +39,10 @@ impl CommandApi {
         }
     }
 
+    pub fn new_from_cffi(accounts: Arc<RwLock<Accounts>>) -> Self {
```

Comment:
we can make this `new()` and remove the other `new()` method. Or call this `new_from_arc` or so. The passed-in value has nothing to do with `cffi` so doesn's make sense to call it `new_from_cffi`.

Reply:
> has nothing to do with cffi

it's only used in the cffi, but sure I'll rename it to the more generic `new_from_arc`